### PR TITLE
Add a recipe to ease installation of Blackfire

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | symfony/symfony-docs#...

### CODE REVIEW

File: blackfireio/blackfire-symfony-meta/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "env": {
+        "#1": "Get your SERVER credentials at https://blackfire.io/my/settings/credentials",
+        "#2": "BLACKFIRE_SERVER_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+        "#3": "BLACKFIRE_SERVER_TOKEN=yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy"
+    },
+    "docker-compose": {
+        "docker-compose.override.yml": {
+            "services": [
+                "blackfire:",
+                "  image: blackfire/blackfire",
```
Comment (lolautruche) [inline blackfireio/blackfire-symfony-meta/1.0/manifest.json:14]: blackfire/blackfire:2

File: blackfireio/blackfire-symfony-meta/1.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                "  env_file: .env.local",
+                "  environment:",
+                "    BLACKFIRE_LOG_LEVEL: 4",
+                "  ports: [8707]"
```
Comment (lolautruche) [inline blackfireio/blackfire-symfony-meta/1.0/manifest.json:18]: 8307

File: blackfireio/blackfire-symfony-meta/1.0/post-install.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+  * Blackfire Agent has been configured in your <comment>docker-compose.yaml</> file
+
+  * Configure your Blackfire <comment>SERVER</> credentials in <info>.env.local</> (get them at https://blackfire.io/my/settings/credentials)
```
Comment (lolautruche) [inline blackfireio/blackfire-symfony-meta/1.0/post-install.txt:3]: We should be more explicit here, not to confuse people with environment credentials

**Review by lolautruche — CHANGES_REQUESTED** [review]

File: blackfireio/blackfire-symfony-meta/1.0/post-install.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+  * Blackfire Agent has been configured in your <comment>docker-compose.yaml</> file
```
Comment (OskarStark) [inline blackfireio/blackfire-symfony-meta/1.0/post-install.txt:1]:
```suggestion
  * Blackfire Agent has been configured in your <comment>docker-compose.override.yaml</> file
```
?

Comment (fabpot) [reply blackfireio/blackfire-symfony-meta/1.0/post-install.txt:1]: Good catch indeed!

File: blackfireio/blackfire-symfony-meta/1.0/post-install.txt (hunk 3 above)
Comment (lolautruche) [reply blackfireio/blackfire-symfony-meta/1.0/post-install.txt:3]: Thinking about it, we'd better keep this link.
Available credentials (for Blackfire environments and personal server credentials) should probably get listed in the Blackfire settings page.
Ping @ChristopheDujarric

**Pull request merged**